data/content/environment_variables.yaml

```
@@ -215,6 +215,21 @@ variables:
     The GitHub deployment payload data as serialized JSON. Only available on builds triggered by a [GitHub Deployment](https://developer.github.com/v3/repos/deployments/).
   modifable: false
   example: "production"
+- name: BUILDKITE_GROUP_ID
+  desc: |
+    The UUID of the [group step](/docs/pipelines/group-step) that the job belongs to. This variable is only available if the job belongs to a group step.
+  modifable: false
+  example: "4a331026-8c9a-4714-aff0-8aa30211a34e"
+- name: BUILDKITE_GROUP_KEY
+  desc: |
+    The value of the `key` attribute of the [group step](/docs/pipelines/group-step) that the job belongs to. This variable is only available if the job belongs to a group step.
+  modifable: false
+  example: "audit-tasks"
+- name: BUILDKITE_GROUP_LABEL
+  desc: |
+    The label/name of the [group step](/docs/pipelines/group-step) that the job belongs to. This variable is only available if the job belongs to a group step.
```

It would be good to avoid the "a/b" syntax here. Looking at the group step, I assume this is the `label` attribute. If so, let's follow the same pattern you used for the `key` attribute.

```suggestion
    The value of the `label` attribute of the [group step](/docs/pipelines/group-step) that the job belongs to. This variable is only available if the job belongs to a group step.
```

Does that work?

---

Not quite - it's either the label attribute, or the name set using `group` - you can read more here: https://bk-docs-pr-1917.onrender.com/docs/pipelines/group-step#group-step-attributes

This was copied from the [BUILDKITE_LABEL](https://bk-docs-pr-1917.onrender.com/docs/pipelines/environment-variables#BUILDKITE_LABEL) docs, which work the same way, but applies to the current job, not the parent group

---

Okay, wishful thinking on my part haha. Let's leave it like you had, with the "that" removed here too.

```suggestion
    The label/name of the [group step](/docs/pipelines/group-step) the job belongs to. This variable is only available if the job belongs to a group step.
```